overlord/snapstate/snapmgr.go

```diff
@@ -84,33 +75,100 @@ func Manager(s *state.State) (*SnapManager, error) {
 	}
 
 	runner.AddHandler("install-snap", m.doInstallSnap)
+	runner.AddHandler("update-snap", m.doUpdateSnap)
 	runner.AddHandler("remove-snap", m.doRemoveSnap)
+	runner.AddHandler("purge-snap", m.doPurgeSnap)
+	runner.AddHandler("rollback-snap", m.doRollbackSnap)
+	runner.AddHandler("set-active-snap", m.doSetActiveSnap)
```

Comment:
Yes, done. I assume we are ok with `s.NewTask("activate-snap", false)` when we deactivate it? Or should I make a handler for  `activate` and `deactivate`?